verification/src/http_server/handlers/verification/solidity/contract_verifier.rs

```diff
@@ -68,13 +70,15 @@ pub(crate) async fn compile_and_verify_handler(
     ))
 }
 
+#[instrument(skip(compilers, verifier, input), level = "debug")]
 async fn compile_and_verify(
     compilers: &Compilers,
     verifier: &Verifier,
     input: &Input<'_>,
 ) -> Result<VerificationSuccess, CompileAndVerifyError> {
     let compiler_output = compilers
         .compile(&input.compiler_version, &input.compiler_input)
+        .in_current_span()
```

Comment:
is this doing anything? like, you've already got a span for whole function, and this call isn't a separate span, what happens if you add this line? O.o